include/crocoddyl/core/activations/quadratic-flat-exp.hpp

```diff
@@ -103,9 +103,8 @@ class ActivationModelQuadFlatExpTpl : public ActivationModelAbstractTpl<_Scalar>
    * @return the activation data
    */
   virtual boost::shared_ptr<ActivationDataAbstract> createData() {
-    boost::shared_ptr<ActivationDataAbstract> data =
-        boost::allocate_shared<ActivationDataAbstract>(Eigen::aligned_allocator<ActivationDataAbstract>(), this);
-    data->Arr.diagonal().fill((Scalar)1.);
+    boost::shared_ptr<Data> data =
```

Comment:
there are missing noalias() in the calcdiffs of the two activations. Could you please add them here as well?

Reply:
`a1`, `alpha_` are scalars.
Those lines are performing a scalar-vector multiplication.
As far I understand, scalars multiplication don't allocate data; therefore, it is not needed `noalias()`

Reply:
@proyan do you agree with that?
(I could modify the lines, but I want to be sure that it is really needed)

Reply:
![image](https://user-images.githubusercontent.com/3601935/104465531-0ade2a80-55ac-11eb-9c1b-32def7641406.png)

This is comment from Eigen. "Scalar are harmless"
This is why I understand that `noalias()` is not needed in those cases.